package-lock.json

```diff
@@ -0,0 +1,10711 @@
+{
+  "name": "solpay-template",
```

Comment:
This file needs to be removed, install deps using `yarn install`, we can not maintain two lock files